requirements_all.txt

```diff
@@ -1816,7 +1816,7 @@ python-songpal==0.12
 python-tado==0.8.1
 
 # homeassistant.components.telegram_bot
-python-telegram-bot==11.1.0
+python-telegram-bot==13.0
```

Comment:
Please update to the latest version 13.1

https://github.com/python-telegram-bot/python-telegram-bot/releases/tag/v13.1